pkg/volume/quobyte/quobyte.go

```diff
@@ -158,6 +158,23 @@ func (plugin *quobytePlugin) newMounterInternal(spec *volume.Spec, pod *api.Pod,
 		return nil, err
 	}
 
+	if pod != nil {
+		// We use the service account if it's not the default service account
+		if len(pod.Spec.ServiceAccountName) != 0 && pod.Spec.ServiceAccountName != "default" {
+			source.User = pod.Spec.ServiceAccountName
+			source.Group = pod.Spec.ServiceAccountName
```

Comment:
are these supposed to map to named users on the node OS? that's not what service account is